server/integration/testsuite/src/test/java/org/infinispan/server/test/security/rest/RESTCertSecurityIT.java

```diff
@@ -77,53 +77,65 @@
     @InfinispanResource("rest-security-cert")
     RemoteInfinispanServer server;
 
-    @Test
-    @WithRunningServer({@RunningServer(name = CONTAINER, config = "testsuite/rest-sec-cert-wr.xml")})
-    public void testSecuredWriteOperations() throws Exception {
-        //correct alias for the certificate
-        put(securedClient(testAlias), keyAddress(KEY_A), HttpStatus.SC_OK);
-        //test wrong authorization, 1. wrong alias for the certificate
-        put(securedClient(test2Alias), keyAddress(KEY_B), HttpStatus.SC_FORBIDDEN);
-        //2. access over 8080
-        put(securedClient(testAlias), keyAddressUnsecured(KEY_B), HttpStatus.SC_UNAUTHORIZED);
-        post(securedClient(testAlias), keyAddress(KEY_C), HttpStatus.SC_OK);
-        post(securedClient(test2Alias), keyAddress(KEY_D), HttpStatus.SC_FORBIDDEN);
-        //get is not secured, should be working over 8080
-        HttpResponse resp = get(securedClient(test2Alias), keyAddressUnsecured(KEY_A), HttpStatus.SC_OK);
-        String content = new BufferedReader(new InputStreamReader(resp.getEntity().getContent())).readLine();
-        assertEquals("data", content);
-        head(securedClient(test2Alias), keyAddressUnsecured(KEY_A), HttpStatus.SC_OK);
-        delete(securedClient(test2Alias), keyAddress(KEY_A), HttpStatus.SC_FORBIDDEN);
-        delete(securedClient(testAlias), keyAddress(KEY_A), HttpStatus.SC_OK);
-        delete(securedClient(testAlias), keyAddress(KEY_C), HttpStatus.SC_OK);
+    static CloseableHttpClient securedTest;
```

Comment:
Because JUnit enforces BeforeClass/AfterClass methods to be static, therefore fields referenced there must also be static